good-enough-practices-for-scientific-computing.tex

```diff
@@ -917,7 +962,7 @@ \subsection*{Version Control Systems}
 
 What the manual process described above requires most is
 self-discipline. The version control tools that underpin our second
-approach---the one all authors use for their projects---don't just
+approach, and the one we use in our projects, don't just
```

Comment:
I prefer the first form, since "the one" in the second form reads to me as "the tools" rather than "the approach"